### PR TITLE
Configure sendgrid

### PR DESCRIPTION
**Pull request by ragurney**
Pull request to enable the ability to send emails in production.

### CODE REVIEW

**Comment by ragurney** [comment]
@rmacklin this is still being tested. I will assign you to this PR once that process is complete.

**Comment by ragurney** [comment]
The review app is able to send email. Ready to test it on the staging app.

File: app.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
     }
   },
   "addons": [
-    "heroku-postgresql"
+    "heroku-postgresql",
+    "sendgrid"
```
Comment (rmacklin) [inline app.json]: It's safer to explicitly declare the free plans: `"heroku-postgresql:hobby-dev"` and `"sendgrid:starter"`

**Comment by rmacklin** [comment]
Other than the one comment, looks good

File: app.json (hunk 1 above)
Comment (ragurney) [inline app.json]: 3f9202a

**Pull request merged**